src/main/webapp/app/localci/build-agents/build-agent-details/build-agent-details/build-agent-details.component.html

```diff
@@ -1,10 +1,25 @@
 <div>
     @if (buildAgent) {
         <div style="padding-bottom: 40px">
-            <div style="display: flex; align-items: center; padding-bottom: 20px">
-                <h3 id="build-agent-heading" jhiTranslate="artemisApp.buildAgents.details"></h3>
-                <span class="h3">:</span>
-                <h3 id="build-agent-name" class="h3" style="margin-left: 20px">{{ buildAgent.name }}</h3>
+            <div style="padding-bottom: 20px" class="d-flex align-items-center justify-content-between">
+                <div class="d-flex align-items-center">
+                    <h3 id="build-agent-heading" jhiTranslate="artemisApp.buildAgents.details"></h3>
+                    <span class="h3">:</span>
+                    <h3 id="build-agent-name" class="h3" style="margin-left: 20px">{{ buildAgent.name }}</h3>
```

Comment:
```suggestion
                    <h3 id="build-agent-name" class="mb-0 ms-3">{{ buildAgent.name }}</h3>
```

Reply:
mb-0 needed, otherwise align-items-center wont work due to the additional margin of h3